src/unix/aix.c

```diff
@@ -768,6 +770,15 @@ int uv_fs_event_start(uv_fs_event_t* handle,
 
   uv__io_start(handle->loop, &handle->event_watcher, POLLIN);
 
+  /* AHAFS wants someone to poll for it to start mointoring.
+   *  so kick-start it so that we don't miss an event in the
+   *  eventuality of an event that occurs in the current loop. */
+  FD_ZERO(&pollfd);
+  FD_SET(fd, &pollfd);
```

Comment:
Can you move these inside the loop? They need to be executed every time before entering select().

Reply:
@bnoordhuis - done.